src/service/xhr-impl.js

```diff
@@ -185,7 +187,8 @@ export class Xhr {
     }
     const viewer = Services.viewerForDoc(this.ampdocSingle_);
     const whenFirstVisible = viewer.whenFirstVisible();
-    if (!viewer.hasCapability('xhrInterceptor')) {
+    const firstPartyResource = urls.cdnProxyRegex.test(new URL(input).origin);
```

Comment:
`URL` is not supported everywhere. Use our helper functions in `url.js`.

Reply:
done. thanks.